pkg/operator/ceph/config/monstore.go

```diff
@@ -55,25 +55,31 @@ type Option struct {
 // Set sets a config in the centralized mon configuration database.
 // https://docs.ceph.com/docs/master/rados/configuration/ceph-conf/#monitor-configuration-database
 func (m *MonStore) Set(who, option, value string) error {
+	logger.Infof("setting %q=%q=%q option to the mon configuration database", who, option, value)
 	args := []string{"config", "set", who, normalizeKey(option), value}
 	cephCmd := client.NewCephCommand(m.context, m.clusterInfo, args)
 	out, err := cephCmd.Run()
 	if err != nil {
```

Comment:
if the setting isn't found (maybe it has already been deleted), will this fail or succeed?

Reply:
The call is idempotent so if you try to remove something that is gone it will not fail.